LexBot/Lexbot-deploy.py

```diff
@@ -160,12 +179,18 @@ def createBot(client, botName):
                                slots[slot_name]['slot_enumberation_values']):
             ScottySlots.createSlotTypeVersion(lex_client, slot_name)
 
-    BlackintentName = "BlackList"
+    BlackIntentName = "BlackList"
     ScottyBlacklistIntent.addPermission(
-        session, BlackintentName, args.region, account_id)
+        session, BlackIntentName, args.region, account_id)
+
+    if ScottyBlacklistIntent.createIntent(session, lex_client, BlackIntentName, BL_slot_list) is True:
+        ScottyBlacklistIntent.createIntentVersion(lex_client, BlackIntentName)
```

Comment:
handle errors here, and get version to use from this - pass it to the initializeBot method